editor/plugins/sprite_frames_editor_plugin.cpp

```diff
@@ -200,20 +228,52 @@ void SpriteFramesEditor::_sheet_spin_changed(double) {
 	split_sheet_preview->update();
 }
 
+void SpriteFramesEditor::_sheet_mode_changed(int mode) {
+
+	switch (mode) {
+		case MODE_GRID:
+			split_sheet_grid_vb->show();
+			split_sheet_pixel_vb->hide();
+			break;
+		case MODE_PIXEL:
+			split_sheet_grid_vb->hide();
+			split_sheet_pixel_vb->show();
+			break;
+		default:
+			// It shouldn't happen
+			break;
+	}
```

Comment:
Maybe you could simplify this part of the code with something like :
```
split_sheet_grid_vb->set_visible(mode==MODE_GRID)
split_sheet_pixel_vb->set_visible(mode==MODE_PIXEL)
```

Reply:
Ok, thanks :)